projects/packages/my-jetpack/_inc/data/evaluation-recommendations/use-evaluation-recommendations.ts

```diff
@@ -42,8 +44,10 @@ const useEvaluationRecommendations = () => {
 				? [ 'anti-spam', 'creator', 'extras', 'stats', 'jetpack-ai' ]
 				: getMyJetpackWindowInitialState( 'lifecycleStats' )?.ownedProducts || []
 		) as JetpackModule[];
-		// We filter out owned modules, and return top 3 recommendations
-		return recommendedModules?.filter( module => ! ownedProducts.includes( module ) ).slice( 0, 3 );
+		// We filter out owned modules, and return the top recommendations
+		return recommendedModules
```

Comment:
Probably we will want to change the backend behavior to return only recommendations we actually want to show.

Reply:
This should be addressed in a follow-up IMO.